test/integration/test_live_backend_reporting.py

```diff
@@ -18,8 +18,6 @@
 
 
 @pytest.mark.skipif(
-    # TODO FIXME Remove the conditions for skipping this on Amazonian fleets when we move fully to GitHub.
-    socket.gethostname().startswith("pb-worker-prod") or
     socket.gethostname().startswith("coverlay-") or
```

Comment:
Should we remove this as well(?) This is something internal right?

Reply:
Yes, I will remove it.